graylog2-web-interface/webpack.config.js

```diff
@@ -14,7 +14,12 @@ const TARGET = process.env.npm_lifecycle_event;
 process.env.BABEL_ENV = TARGET;
 
 const BABELRC = path.resolve(ROOT_PATH, '.babelrc');
-const BABELLOADER = 'babel-loader?cacheDirectory&extends=' + BABELRC;
+const BABELOPTIONS = {
```

Comment:
While I am not against that, I don't think we should do it here, as it is changing parsing semantics beyond the scope of this PR.